images/ci-kubernetes-e2e-kubeadm/Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 0.4
+VERSION = 0.5
```

Comment:
[I](https://github.com/kubernetes/test-infra/blob/master/images/bigquery/Makefile#L15) [would](https://github.com/kubernetes/test-infra/blob/master/images/ci-kubernetes-e2e-kubeadm/Makefile#L15) [beg](https://github.com/kubernetes/test-infra/blob/master/images/gcloud/Makefile#L15) [to](https://github.com/kubernetes/test-infra/blob/master/images/heapster/Makefile#L15) [differ](https://github.com/kubernetes/test-infra/blob/master/images/pull-test-infra-gubernator/Makefile#L15). [:-)](https://github.com/kubernetes/test-infra/blob/master/images/pull_kubernetes_bazel/Makefile#L15)

It looks like only one out of seven of the images in `test-infra/images` is using the stamp-commit pattern  so far ([e2e-prow](https://github.com/kubernetes/test-infra/blob/master/images/e2e-prow/Makefile#L15)).